### ISSUE TITLE
Need help with some severity level instead of cvss level changes to HTML code.

### ISSUE THREAD

**Issue by Gray-0men** [Syslifters/sysreptor#302]
![image](https://github.com/user-attachments/assets/ae30e388-8f7c-4ec0-b06d-b3170f505ae4)

Currently trying to put in the severity predefined field on our templates, since I just noticed it, but I'm having a hard time figuring out how to do a finding.severity.level like change to the HTML code instead of using finding.cvss.level as it's doing now making my chart and "Identified Vulnerabilities" section look misleading with the colors. 
I'm hoping there's a quick fix in this line of code
`<tr v-for="finding in findings" class="table-row-link" :class="'risk-bg-' + finding.cvss.level">`
to become 
`<tr v-for="finding in findings" class="table-row-link" :class="'risk-bg-' + finding.severity.level">`
or something similar, basically I'm trying to find information on how to use the new enum severity level instead of cvss derived severity level in some HTML code. Noticing other area's where I could replace other finding.cvss.level pieces of code too that I might have to change as well.

Any help or clarity on how to properly use the severity finding field data in the HTML code would be cool,
Thank you for the assist

**Comment by Gray-0men** [comment]
`finding.cvss.level`
to
`finding.severity.value`

Figured it out, if anyone else was looking

**Comment by aronmolnar** [comment]
Correct. The predefined `severity` field is an enum which has a `value` and a `label` attribute.  
See also: https://docs.sysreptor.com/designer/field-types/#enum
